sql-cli/tests/config/test/configuration.yml

```diff
@@ -22,7 +22,7 @@ connections:
     schema: pagila
     login: postgres
     password: $POSTGRES_PASSWORD
-    port: $POSTGRES_PORT
+    port: 5432
```

Comment:
Is this change intended, @pankajkoti ?

Reply:
yes, the change is intentional. A validate test was failing on my local saying not a valid integer.

Reply:
@pankajkoti this seems to be an issue to your local environment/setup. I am not having the issue and using `POSTGRES_PORT=5433` in my `.env` file. Now I am unable to use the same db. I would be in favor of reverting this change, if that's okay for you both? @tatiana

Reply:
It is an issue if the environment variable in not set

Reply:
okay, will add back the env var.